app/konnect/getting-started/app-registration.md

```diff
@@ -98,6 +100,12 @@ reference ID, and description. The reference ID must be unique.
 ## Summary and next steps
 
 In this topic, you:
-* Enabled application registration for the service `example_service`.
-* From the Dev Portal, created an application and registered it against the `example_service`.
-* Generated an API key for the application and made a call using this key.
+* Enabled application registration for the a service
+* Created an application through the Dev Portal and registered it against your service
+* Generated an API key for the application and made a call using this key
+
+For next steps, check out some of the other things you can do in
```

Comment:
When I was working through this, I wasn't sure why it didn't link out to [Import Kong Gateway Entities into Konnect Cloud](https://deploy-preview-4073--kongdocs.netlify.app/konnect/getting-started/import/) since that's the next section in this guide. Should that page be moved to a different section or called out somewhere so users know when they should go to that page?